src/foam/u2/view/DAOtoFObjectArrayView.js

```diff
@@ -57,7 +55,9 @@ foam.CLASS({
 
       this.addClass();
 
-      this.start(this.daoView, { data: this.dao }).end();
+      this.tag(this.STACK);
```

Comment:
I don't think you need these three lines. Just use the original. And if you're going to remove the 'stack' export, you can also remove the stack property.

Reply:
got it.